src/main/java/io/reactivex/Flowable.java

```diff
@@ -10345,6 +10345,17 @@ public final Disposable forEachWhile(final Predicate<? super T> onNext, final Co
      * is subscribed to. For this reason, in order to avoid memory leaks, you should not simply ignore those
      * {@code GroupedPublisher}s that do not concern you. Instead, you can signal to them that they may
      * discard their buffers by applying an operator like {@link #ignoreElements} to them.
+     * <p>
+     * By default, a {@link GroupedFlowable} buffers only {@code Flowable.BUFFER_SIZE} items to emit.
+     * So, it is important that a subscription to consume these items begin as soon as the {@link GroupedFlowable}s are created.
+     * If not, it may result in groups upstream not being consumed.
+     * Typically {@code flatMap} or {@code concatMapEager} can be used to ensure all {@link GroupedFlowable} items are consumed.
+     * <p>
+     * Note that, while using {@link #flatMap(Function, int)} or {@link #concatMapEager(Function, int, int)} operators,
+     * we need to ensure that the maximum concurrency parameter is a value greater than
```

Comment:
> "we need to ensure"

This wording doesn't really fit into a JavaDoc.

Reply:
Noted. Thanks for the tip.